amd_build/build_pytorch_amd.py

```diff
@@ -0,0 +1,47 @@
+import shutil
+import subprocess
+import os
+
+cwd = os.getcwd()
```

Comment:
It would be more robust to use `__file__` to determine where the script lives; otherwise this script is sensitive to the particular directory you run it in.